Contentless/Contentless.csproj

```diff
@@ -20,6 +21,12 @@
         <PackageReference Include="Newtonsoft.Json" Version="13.0.1">
             <PrivateAssets>All</PrivateAssets>
         </PackageReference>
+        <PackageReference Include="Microsoft.Build" Version="17.3.2">
```

Comment:
What exactly is this used for? Can't we just prase project files as xml and iterate them ourselves? Is this more stable/secure?

Reply:
I have written about this in main message. It's first try, I don't do deep analysis. `Microsoft.Build` are more clear, because 'knows' about .csproj. But has problem with concurrency with local .Net SDK. If you know way to fix this, better `Microsoft.Build`, not - usually XML.